funnel/views/event.py

```diff
@@ -209,4 +209,6 @@ def event_checkin(profile, space, event, participant):
     a.checked_in = checked_in
     db.session.add(a)
     db.session.commit()
+    if request.method == 'POST':
```

Comment:
that is fine for ajax, but Android's HTTP calls don't have the XHR header. I'll have to add it manually, unless we define a different endpoint for android (which doesn't make sense).